carbonmark-data/components/pages/DetailPage/index.tsx

```diff
@@ -6,12 +9,47 @@ export default function DetailPage(props: {
   pageTitle: string;
   chartTitle: string;
   overview: string;
+  insights?: {
+    content: string;
+    source: "ai" | "human";
```

Comment:
not sure what to categorize these as…

Reply:
From what @theprofessora said Insights are not ready yet. 
Lets say we adjust at the end of the project.